package.json

```diff
@@ -133,7 +134,7 @@
     "build-storybook": "build-storybook -c stories",
     "flow": "flow",
     "flow-typed-rebuild": "rm -rf flow-typed/ && flow-typed install && flow-typed install --rootDir packages/@cruise-automation/hooks && flow-typed install --rootDir packages/@cruise-automation/button && flow-typed install --rootDir packages/@cruise-automation/tooltip && flow-typed install --rootDir packages/regl-worldview",
-    "docs": "DEV_SERVER=true webpack-dev-server",
+    "docs": "NODE_OPTIONS='--max-old-space-size=4096' DEV_SERVER=true webpack-dev-server",
```

Comment:
Curious did you hit some performance issue in the open source version or just thought it's a good idea to add `--max-old-space-size`?

Reply:
i mention it in the pr description-- i was hitting errors without this option